tests/test_bar.py

```diff
@@ -2,7 +2,7 @@
 
 from rich.bar import Bar
 
-from render import render
+from .render import render
```

Comment:
Curious why does this not fail with "attempted relative import with no known parent package" ?

There's some code at the bottom of some of these files which generates the expected output. I run them with "python test_bar.py", which will break if there are relative imports.